ml-agents/mlagents/trainers/stats.py

```diff
@@ -279,14 +313,19 @@ def write_stats(self, step: int) -> None:
 
     def get_stats_summaries(self, key: str) -> StatsSummary:
         """
-        Get the mean, std, and count of a particular statistic, since last write.
+        Get the mean, std, count, sum and aggregation method of a particular statistic, since last write.
+
         :param key: The type of statistic, e.g. Environment/Reward.
-        :returns: A StatsSummary NamedTuple containing (mean, std, count).
+        :returns: A StatsSummary NamedTuple containing (mean, std, sum, count, aggregation).
```

Comment:
```suggestion
        :returns: A StatsSummary containing summary statistics.
```
(just make this a bit more future-proof)